packages/terra-core-docs/src/terra-dev-site/doc/switch/AccessibilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,178 @@
+
+- [**1.4.3 Contrast(Minimum)**](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) - Supports
+  - Terra switch provides components that, by default, meet color contrast requirements.
+  - Engineers must work with content creators to implement switch so that switch text (visual label) color and the background color meet appropriate color contrast ratios.
+  - Content creators must ensure that the switch text color meets the appropriate contrast ratio against background color(s) based on text size of the visual label text. Contrast ratios should be above 4.5:1 for normal text and 3:1 for bold and large text.
+
+- [**2.4.3 Focus Order**](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html) — Supports
+  - Terra provides the ability for actionable components to receive focus in the order they are coded.
+  - Engineers must ensure that the focus order is expected and preserves page meaning. Code order should follow the logical reading order of the user. Though a TABINDEX attribute should not be required to ensure keyboard use, if used, never use TABINDEX higher than 0.
+  - Content creators should convey to the engineers the logical reading order of the page.
+
+- [**2.5.2 Pointer Cancellation**](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation)  — Supports 
+  - Terra components use the up event to perform actions.
+  - Engineers must ensure actionable controls always take on the up event.
+  
+- [**3.2.1 On Focus**](https://www.w3.org/WAI/WCAG21/Understanding/on-focus) —Supports 
+  - Terra components do not cause a change of context on focus by default.
+  - Engineers must ensure that no unexpected change of context happens when interactive elements receive focus.
+  - Content creators must inform engineers that switch should not cause any change while on focus.
+
+- [**3.2.4 Consistent Identification**](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification) - Supports
+  - Terra provides the ability for elements to be consistently identified.
+  - Engineers must ensure that elements used for the same function are identified consistently.
+  - Content creators must ensure any icons used are used to represent the same function consistently. Additionally, the alternate content used to describe icons must be consistent for users to understand their function predictably.
+
+- [**4.1.1 Parsing**](https://www.w3.org/WAI/WCAG21/Understanding/parsing.html) - Supports
+  - Terra components are tested and validated before release to ensure proper code parsing.
+  - Engineers must ensure their code is valid HTML.
+
+- [**4.1.2 Name,Role,Value**](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) - Supports
+  - Terra provides the ability for Name, Role, and Value attributes to be programmatically determined so as to be read by screen readers.
```

Comment:
```suggestion
  - Terra provides the ability for `name`, `role`, and `value` attributes to be programmatically determined so that they can be read by screen readers.
```